compiler/rustc_trait_selection/src/traits/effects.rs

```diff
@@ -226,6 +232,105 @@ fn evaluate_host_effect_from_item_bounds<'tcx>(
     }
 }
 
+fn evaluate_host_effect_from_builtin_impls<'tcx>(
+    selcx: &mut SelectionContext<'_, 'tcx>,
+    obligation: &HostEffectObligation<'tcx>,
+) -> Result<ThinVec<PredicateObligation<'tcx>>, EvaluationFailure> {
+    match selcx.tcx().as_lang_item(obligation.predicate.def_id()) {
+        Some(LangItem::Destruct) => evaluate_host_effect_for_destruct_goal(selcx, obligation),
+        // Some(LangItem::Fn | LangItem::FnMut | LangItem::FnOnce) =>
```

Comment:
either yeet this, or make it syntactically valid and add a `// FIXME(const_effects)`

Reply:
oh right, was gonna add those lang items too in a follow-up but got distracted.
will yeet.